release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1892,7 +1892,7 @@ With this release, availability sets are no longer modified after creation, allo
 
 * Previously, if an application did not use the Path MTU discovery (PMTUD) mechanism while sending UDP packets larger than the network MTU, an issue with the `OVN` package caused the a packet to be dropped while fragmenting the packet. With this release, the `OVN` package has been fixed so that large UDP packets are properly fragmented and sent over the network. (link:https://issues.redhat.com/browse/OCPBUGS-43649[OCPBUGS-43649])
 
-* Previously, a pod with an IP address in an OVN-Kubernetes `Localnet` network was unreachable by other pods that ran on the same node but used the default network for communication. Communication between pods on different nodes was not impacted by this communication issue.  With this release, communication between a `Localnet` pod and a default network pod that both run on the same node is possible. (link:https://issues.redhat.com/browse/OCPBUGS-43004[OCPBUGS-43004])
+* Previously, a pod with an IP address in an OVN-Kubernetes `Localnet` network that was plugged into a `br-ex` interface bridge was out of reach by other pods on the same node, but used the default network for communication. The communication between pods on different nodes was not impacted. With this release, the communication between a `Localnet` pod and a default network pod running on the same node is possible, however the IP addresses that are used in the `Localnet` network must be within the same subnet as the host network. (link:https://issues.redhat.com/browse/OCPBUGS-43004[OCPBUGS-43004])
```

Comment:
Since we're at it, can we also modify the first sentence? It's not entirely correct, I overlooked it at first.

I'd say: `Previously, a pod with a secondary interface in an OVN-Kubernetes Localnet network...`

Thanks!